StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -30,6 +30,12 @@ class StFttClusterMaker: public StMaker {
     int  Finish();
     int  Make();
 
+    void SetTimeCut( int mode, int min, int max ) {
```

Comment:
If one sets the mode value to -1 via this function, then min and max are meaningless. I suggest adding some defaults for those, e.g. min=-999, max=999.